tasks/components/user_input/Textarea.tsx

```diff
@@ -0,0 +1,20 @@
+import { tw } from '@helpwave/common/twind'
+
+type TextareaProps = {
+  headline : string,
+  placeholder : string,
+  message : string,
+  id : string
```

Comment:
All of these parameters should be optional

Reply:
Would be nice to have an min and max length parameter

Reply:
pass `{...props}` to the input element, see [this](https://github.com/helpwave/web/blob/main/tasks/components/user_input/Input.tsx) for a reference

Reply:
thats a good idea